versioned_docs/version-3.0.0-beta/organization/personal-accounts.md

```diff
@@ -0,0 +1,28 @@
+---
+id: personal accounts
+title: Personal Accounts
+sidebar_label: Personal Accounts
+slug: /organization/personal-accounts
+---
+
+## Personal Accounts
+
+When you create a new account with Spheron, you are creating a Personal Account. A Personal Account is your own, individual account, registered with the Spheron platform and can be upgraded and can act as an admin account for your organization. 
+
+## Understanding the Personal Account and Pro Plan 
```

Comment:
We don't do this. Though we have a personal account, we create a particular organization that works like any other. It's a bit different from Vercel as vercel enforces personal accounts as a completely different organization.

So there is no problem with users who wants to upgrade to Pro plan for their organization created when they joined Spheron